scripts/globals/mobskills/mijin_gakure.lua

```diff
@@ -1,22 +1,17 @@
 ---------------------------------------------------
 -- Mijin Gakure
 ---------------------------------------------------
-
+require("scripts/globals/monstertpmoves")
 require("scripts/globals/settings")
 require("scripts/globals/status")
-require("scripts/globals/monstertpmoves")
-
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-    if (mob:getMobMod(dsp.mobMod.SCRIPTED_2HOUR) == 1) then
-        return 0
-    elseif (skill:getParam() == 2 and math.random() <= 0.5) then -- not always used
+    if skill:getParam() == 2 and math.random() <= 0.5 then -- not always used
```

Comment:
isn't this exact thing handled by a feature of the new mixin?

Reply:
yeah, the chance param determines whether mob will use a 2 hr or not. i didn't know what all these skill:getParam() things were in the 2hr code, so I left 'em in.  Can I strip them all out?  (I haven't written the mixin yet when I edited these files.)

Reply:
make'em dead :ghost: 
they are all old leftovers form when 2hrs had to be called in core as a mobskill. flags to prevent mob from using it via tp, parans that said when they were allowed..all that can move to the mixin

Reply:
done